scripts/get_changed_pipeline_worklow_test_args.sh

```diff
@@ -22,34 +22,34 @@ function pipeline_to_args() {
       echo Arrays -a Single ${common_args};;
     MultiSampleArrays)
       echo Arrays -a Multi ${common_args};;
-    ExomeGermlineSingleSample)
-      echo GermlineSingleSample -d Exome ${common_args};;
-    ExomeReprocessing)
-      echo Reprocessing -d Exome ${common_args};;
-    JointGenotyping)
-      echo JointGenotyping -d Exome ${common_args};
-      echo JointGenotyping -d WGS --env ${env} -t Plumbing -b develop ${uncached};;
+    #ExomeGermlineSingleSample)
```

Comment:
If these are commented out, then an unsupported test will result in empty output. Is that what we want to do? Vs generating an error or printing a message, etc.

Reply:
My thought was that we would bring them back in as the pipelines were brought over. We can do it a different way if you prefer

Reply:
It's OK, I guess we'll notice if things do nothing vs forcing it to fail.